spicy/toolchain/src/compiler/codegen/unit-builder.cc

```diff
@@ -163,6 +163,7 @@ Type CodeGen::compileUnit(const type::Unit& unit, bool declare_only) {
     add_hook("0x25_init", {});
     add_hook("0x25_done", {});
     add_hook("0x25_error", {});
+    add_hook("0x25_print", {});
```

Comment:
We'll need a doc update for this as well eventually.

Reply:
I added docs for `%print`, do you mean something else?

Reply:
That one is perfect, missed it.